Libplanet.Tests/Net/Protocols/TestTransport.cs

```diff
@@ -248,10 +252,14 @@ public void BroadcastTestMessage(Address? except, string data)
         public void BroadcastMessage(Address? except, Message message)
         {
             var peers = Protocol.PeersToBroadcast(except).ToList();
+            var peersString = peers.Aggregate(
+                new StringBuilder(),
+                (builder, peer) => builder.Append($"{peer.Address.ToHex()}, ")).ToString();
```

Comment:
Can't the same thing be easily achieved using [`string.Join()` method][1]?

[1]: https://docs.microsoft.com/en-us/dotnet/api/system.string.join?view=netstandard-2.0